### ISSUE TITLE
Error with printing nodes with utf-8 characters in python 2.7

### ISSUE THREAD

**Issue by owen-jones-gen** [althonos/pronto#6]
Using the test.obo

```
[Term]
id: ONT0:ROOT
name: °
```

when I run the the following python script with python 2.7 and pronto 0.4.0:

```
import pronto

obo = pronto.Ontology("./test.obo")
for node in obo:
    print(node)
```

I get the error message

```
Traceback (most recent call last):
  File "test.py", line 5, in <module>
    print(node)
  File "/local/scratch/PopulationAnalysis/env-phos/lib/python2.7/site-packages/pronto/term.py", line 55, in __repr__
    return "<{}: {}>".format(self.id, self.name)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xb0' in position 0: ordinal not in range(128)
```

I'm guessing that the __repr__ should have some of the strings marked as unicode.

**Comment by althonos** [comment]
Hi ! Thanks for the feedback, as I'm not really using Python 2 that's the kind of issues that can stay unseen for quite a long time. 

I also fixed Term.obo and Ontology.obo so that failed as well with an utf-8 character.

**Comment by althonos** [comment]
Okay, so that's now fixed in v0.4.2, and I added a unittest TestCase for this issue.

**Comment by owen-jones-gen** [comment]
Great, thanks. I look forward to trying it out next time I upgrade.
